src/ucp/core/ucp_context.c

```diff
@@ -312,11 +312,11 @@ static ucs_config_field_t ucp_config_table[] = {
    ucs_offsetof(ucp_config_t, ctx.proto_enable), UCS_CONFIG_TYPE_BOOL},
 
   /* TODO: set for keepalive more reasonable values */
-  {"KEEPALIVE_INTERVAL", "0",
+  {"KEEPALIVE_INTERVAL", "60s",
```

Comment:
@hoopoepg will there be keepalive operation if all ep's are without error handling support, for example MPI case?

Reply:
ep_check will not be called, kernel based KA (TCP transport) will be enabled

Reply:
ok, i see that if all ep's are w/o ERR mode, KA (by ep_check) is not done in UCP